ci/test_wheel.sh

```diff
@@ -9,9 +9,9 @@ package_dir=$2
 mkdir -p ./dist
 RAPIDS_PY_CUDA_SUFFIX="$(rapids-wheel-ctk-name-gen ${RAPIDS_CUDA_VERSION})"
 
-# echo to expand wildcard before adding `[extra]` requires for pip
```

Comment:
There was a case where the wildcard did not match the .whl files on disk and 'echo' simply echo'd the wildcard pattern, which made `pip` generate a misleading error message.  Using 'ls' will result in the script erroring out with a clear message about the missing/mis-named files.

Reply:
Odd, but I'm fine with the change if it works.

Reply:
I agree with this change, I see the problem with shell wildcard expansion.